index.js

```diff
@@ -32,6 +32,21 @@ function refresh(win) {
 	}
 }
 
+function inspectElements() {
+	const win = BrowserWindow.getFocusedWindow();
+
+	if (win) {
+		if (win.webContents.isDevToolsOpened()) {
+			win.devToolsWebContents.executeJavaScript('DevToolsAPI.enterInspectElementMode()');
+		} else {
+			win.webContents.on('devtools-opened', () => {
+				win.devToolsWebContents.executeJavaScript('DevToolsAPI.enterInspectElementMode()');
```

Comment:
This is repeated twice and could be abstracted into an inline arrow function below line 36.